lib/Relation.php

```diff
@@ -59,11 +59,14 @@ private function table(): Table
     }
 
     /**
+     * @param string|array<string> $columns
+     *
      * @return Relation<TModel>
      */
-    public function select(string $columns): Relation
+    public function select(string|array $columns): Relation
     {
-        $this->options['select'] = $columns;
+        $this->options['select'] ??= [];
+        $this->options['select'] = array_merge((array) $this->options['select'], (array) $columns);
```

Comment:
Bug: These arrays have numeric keys and therefore the new select columns will be appended to the old ones with an array_merge() implementation. So if you do `Author::select('A')->select('A')` you will get the `SQL: SELECT A,A FROM Authors` when you expect `SELECT A FROM Authors`.

Reply:
PRs welcome, but please demonstrate with a failing test case.